runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java

```diff
@@ -133,9 +137,9 @@ public void onConfigChanged(ConfigChanged event)
 	public void onWidgetHiddenChanged(WidgetHiddenChanged widgetHiddenChanged)
```

Comment:
you said yourself onWidgetHiddenChanged is throwing lots of NPE's

Reply:
It wasn't before, so is this an issue related to injector?

Reply:
not exactly sure as I wasn't the one that implemented the red highlighting in cc